libr/cons/pal.c

```diff
@@ -54,6 +54,9 @@ static struct {
 	{ "func_var_addr", r_offsetof (RConsPrintablePalette, func_var_addr), r_offsetof (RConsPalette, func_var_addr) },
 	{ "widget_bg", r_offsetof (RConsPrintablePalette, widget_bg), r_offsetof (RConsPalette, widget_bg) },
 	{ "widget_sel", r_offsetof (RConsPrintablePalette, widget_sel), r_offsetof (RConsPalette, widget_sel) },
+	{ "bg_true", r_offsetof (RConsPrintablePalette, bg_true), r_offsetof (RConsPalette, bg_true) },
```

Comment:
uhm, no theme will use those new keys, i would rather reuse some other fields that we have, even if thoses have no bg color set, otherwise you'll have to update all the themes

Reply:
Yes you are right. Since the color is based on the permissions. I will just reuse the colors used in the visual method.